core/src/test/scala/kafka/common/InterBrokerSendThreadTest.scala

```diff
@@ -134,36 +129,33 @@ class InterBrokerSendThreadTest {
 
     val clientRequest = new ClientRequest("dest", request, 0, "1", 0, true, requestTimeoutMs, handler.handler)
 
-    EasyMock.expect(networkClient.newClientRequest(
-      EasyMock.eq("1"),
-      EasyMock.same(handler.request),
-      EasyMock.anyLong(),
-      EasyMock.eq(true),
-      EasyMock.eq(requestTimeoutMs),
-      EasyMock.same(handler.handler)))
-      .andReturn(clientRequest)
+    when(networkClient.newClientRequest(
+      ArgumentMatchers.eq("1"),
+      same(handler.request),
+      anyLong(),
+      ArgumentMatchers.eq(true),
+      ArgumentMatchers.eq(requestTimeoutMs),
+      same(handler.handler)))
+      .thenReturn(clientRequest)
 
-    EasyMock.expect(networkClient.ready(node, time.milliseconds()))
-      .andReturn(false)
+    when(networkClient.ready(node, time.milliseconds()))
+      .thenReturn(false)
 
-    EasyMock.expect(networkClient.connectionDelay(EasyMock.anyObject(), EasyMock.anyLong()))
-      .andReturn(0)
+    when(networkClient.connectionDelay(any[Node], anyLong()))
+      .thenReturn(0)
 
-    EasyMock.expect(networkClient.poll(EasyMock.anyLong(), EasyMock.anyLong()))
-      .andReturn(new util.ArrayList())
+    when(networkClient.poll(anyLong(), anyLong()))
+      .thenReturn(new util.ArrayList[ClientResponse]())
 
-    EasyMock.expect(networkClient.connectionFailed(node))
-      .andReturn(true)
+   when(networkClient.connectionFailed(node))
+      .thenReturn(true)
 
-    EasyMock.expect(networkClient.authenticationException(node))
-      .andReturn(new AuthenticationException(""))
-
-    EasyMock.replay(networkClient)
+    when(networkClient.authenticationException(node))
+      .thenReturn(new AuthenticationException(""))
 
     sendThread.enqueue(handler)
     sendThread.doWork()
 
-    EasyMock.verify(networkClient)
```

Comment:
There's a loss of coverage here, since we can't assume that all the methods we mocked actually get called without a mockito `verify`. Do we think that's OK? This is really a general comment.

Reply:
Yeah it's tricky to get the exact same coverage with Mockito. I've tried to verify all the important calls, especially on the component being tested, and whenever possible verify all the mocks.